src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    Square(String color, int side) {
```

Comment:
The constructor of the class should be public to allow instantiation from other packages.